Automattic/wp-calypso
# List Management: add feed and site listing

This PR adds an infinite-scrollable listing of existing feeds and sites in a Reader list (similar to Manage Followed Sites).

![image](https://cloud.githubusercontent.com/assets/17325/12169961/9d6d3058-b534-11e5-93ec-3c1a9ca42232.png)

Fixes #2106.

### To test

Open an existing list using the 'Manage' button next to the list in the Reader sidebar.

Ensure that the sites and feeds show in the Sites tab match those currently shown for the list on wordpress.com.

Old version (currently in production):

<img width="695" alt="screen shot 2016-01-13 at 11 50 31" src="https://cloud.githubusercontent.com/assets/17325/12279446/f9cb7948-b9eb-11e5-9fab-437272b4438a.png">

New version:

<img width="766" alt="screen shot 2016-01-13 at 11 50 57" src="https://cloud.githubusercontent.com/assets/17325/12279450/fe9e76f0-b9eb-11e5-88c5-983bdcf45b47.png">
